src/fn/convert-contact-forms.js

```diff
@@ -19,7 +19,6 @@ const convertContactForm = (forms) => {
 
   return convertForms(environment.pathToProject, 'contact', {
     enketo: true,
-    force_data_node: 'data',
```

Comment:
Now (with the new pyxform) the name of the primary instance node is `data` by default. So we do not have to do anything special for contact forms.